db/migrate/20210624033123_create_keywords.rb

```diff
@@ -0,0 +1,19 @@
+class CreateKeywords < ActiveRecord::Migration[6.1]
+  def change
+    create_table :keywords do |t|
+      t.string :name, null: false
+
+      t.references :user, index: true, null: false
```

Comment:
does adding `foreign_key: true` here work?

Reply:
Tested: Yes, it works well! :) 
Code updated ;-)